model/Core/Classes/PackageVerificationCode.md

```diff
@@ -18,19 +18,21 @@ This identifier enables a recipient to determine if any file in the original pac
 
 Algorithm:
 
+    hashValue = 0
     templist = empty list of strings
 
-    for all files in the package
-        if file is an "excludes" file
+    for all files in the package {
+        if file is a packageVerificationCodeExcludedFile
             skip it /* exclude SPDX analysis file(s) */
         else
-            append algorithm(contents of file) to templist
+            append "algorithm(file)/n" to templist
+        }
 
     sort templist in ascending order by value
-
-    valueslist = join templist values     /* ordered sequence with no separators */
+    
+    valueslist = remove "/n"s from templist  /* ordered sequence with no separators */
```

Comment:
No, there should not be anything to remove. You simply join different elements.

Reply:
This introduces variability on how the join is implemented, and in which language, and which underlying runtimes the language is drawing on. 

See: https://stackoverflow.com/questions/914233/what-is-the-underlying-data-structure-for-python-lists, https://www.geeksforgeeks.org/arraylist-in-java/, ...

Since we're hashing  contents - the implementations matter. 

This was the trap we fell into in 1.1 --> 1.2 - and we should not be introducing it again, esp. for a field that is just meant for compatibility with 2.3.  That this was not discussed before implementing, and could have saved everyone time if it was, especially this late in the cycle, when we're all stressed, and folded into a "pdf formatting change".